### ISSUE TITLE
Scalar market congratulations box

### ISSUE THREAD

**Issue by KadenZipfel** [protofire/omen-exchange#1451]
The congratulations box used for categorical markets doesn't work super well with scalar markets. Instead, it could use a bit of a rewrite. @pimato thoughts?

Photo for reference:
![image](https://user-images.githubusercontent.com/30579067/102146874-9bdfab80-3e1e-11eb-84aa-600d103bc756.png)

**Comment by pimato** [comment]
why are there two outcomes the winning one? long and short?

**Comment by KadenZipfel** [comment]
> why are there two outcomes the winning one? long and short?

Yeah, if the final answer is between the upper and lower bound, technically both long and short are winning outcomes, just paid out in different amounts. So it's a bit different, especially considering you could be at a 99% loss and still have a payout and thus have this congratulations box
